schema/data/prod/fuel.sql

```diff
@@ -92,9 +92,9 @@ values
   -- Flared Natural Gas was previously named Flared Natural Gas CH4
   -- It was renamed and the swrs_fuel_mapping_id was updated as applicants for the CIIP 2019 applications
   -- were instructed to report an amount of natural gas under this fuel
-  (86,'Flared Natural Gas','m3', 'active', 95),
+  (86,'Flared Natural Gas','m3', 'archived', 95),
```

Comment:
I am thinking that we should add a `comments` column to the `fuel` table, to describe why those fuels are archived (and that natural gas should be used instead). The comment above for Flared Natural Gas CH4 would also go in that column.
This wouldn't be intended to be displayed, but as a metadata so that anyone looking at this table in the database can see what's up with those archived fuels.